### PR TITLE
{math}[GCCcore/6.4.0] METIS v5.1.0

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/METIS/METIS-5.1.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['76faebe03f6c963127dbb73c13eab58c9a3faeae48779f049066a21c087c5db2']
+
+# We use 32bit for indices and 64bit for content
+patches = ['METIS-5.1.0-use-doubles.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/m/METIS/METIS-5.1.0-GCCcore-6.4.0.eb]: @JackPerdue One tiny remark: please don't hardcode the version here, use the `%(version)s` template:

```python
patches = ['METIS-%(version)s-use-doubles.patch']

**Comment by JackPerdue** [comment]
I don't know why --update-pr pulled in CMake... its the same 3.9.1 as the one already in devel.

**Comment by boegel** [comment]
@JackPerdue Probably because you combined `--update-pr` with `--robot`, which makes it pull in missing deps (that are not in your PR branch yet).

Nothing to worry about as long as there are no different to the `CMake` easyconfig.

It does confuse GitHub though, it shouldn't be showing the CMake in the diff, but we can check easily with current `develop` via `eb --review-pr`...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/673edd9bb8945d38819a75c635968cfc for a full test report.

**Comment by boegel** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**